devops/infrastructure/chain-spec-configuration.yml

```diff
@@ -16,12 +15,23 @@
 
 - name: Copy secret, auth and start joystream-node service for validators
   hosts: validators
+  gather_facts: no
 
   roles:
     - validators
 
 - name: Configure RPC service and start it
   hosts: rpc
+  gather_facts: no
 
   roles:
     - rpc
+
```

Comment:
The scope of work of this playbook is now more than what it's name suggests. So it should probably get a more "general" name to reflect the tasks it is performing.

Reply:
Changed to `chain-spec-pioneer.yml`